docs/source/user/cap-alerts.rst

```diff
@@ -0,0 +1,139 @@
+.. _cap-alerts:
+
+Publishing CAP Alerts to a wis2box
+==================================
+
+Overview
+--------
+The Common Alerting Protocol (CAP) is a simple but general format for exchanging all-hazard emergency alerts and public warnings over all kinds of networks, see the full specification `here <https://docs.oasis-open.org/emergency/cap/v1.2/CAP-v1.2.html>`_.
+
+This section provides guidance one how one can use the `CAP Composer <https://github.com/wmo-raf/cap-composer>`_ to automate the publishing of CAP alerts to a wis2box.
+
+Requirements
+------------
+In addition to a running wis2box, you will need to install, configure, and run the CAP Composer.
+
+.. note::
+   For information on how to do this, please consult the `CAP Composer documentation <https://nmhs-cms.readthedocs.io/en/stable/_docs/Manage-CAP-Alerts.html>`_.
+
+Process Outline
+---------------
+For automated publishing of CAP alerts to a wis2box, we will need to perform the following steps:
+
+1. *Dataset Creation:* Configure the dataset in the wis2box to store the CAP alerts.
+2. *MQTT Configuration:* Configure your wis2box broker details in the CAP Composer.
+
+Dataset Creation
+----------------
+Firstly, there must be a dataset in the wis2box for the CAP alerts to be stored. To create a dataset, simply navigate to the 'Dataset Editor' page in the wis2box-webapp, available on your host at `http://<your-public-ip>/wis2box-webapp`.
+
+.. note::
+   For more information on how to create a dataset, please see the :ref:`adding-datasets` section of the wis2box setup guide.
+
+When creating a new dataset for CAP alerts, ensure that the **weather/advisories-warnings** template is selected:
```

Comment:
please mention here that weather/advisories-warnings template will use the "CAPMessageData"-data-plugin and make a reference to:
data-pipeline-plugins.rst#wis2box-data-cap-message-capmessagedata